test/test_nn.py

```diff
@@ -6490,6 +6490,10 @@ def test_affine_grid_3d(self):
                 self.assertEqual(input_cpu.grad, input_gpu.grad)
 
     def test_channel_shuffle(self):
+        device_list = ['cpu']
```

Comment:
you can move this test to [`TestNNDeviceType`](https://github.com/pytorch/pytorch/blob/7823b3501538ddfeab0ebeb79bdd444212715202/test/test_nn.py#L7928) to parameterize over device rather than using a `device_list` here